### ISSUE TITLE
False positives on `suspend/network_after_suspend_auto`

### ISSUE THREAD

**Issue by kissiel** [canonical/checkbox#1056]
### Bug Description

The aforementioned job fails with the enigmatic output, see below.

### To Reproduce

Running virtually any TP that has the problematic job and its base sibling included reproduces it.

### Environment

https://test-observer.canonical.com/#/debs/26330

### Relevant log output

```shell
connected
WARNING: systemd-networkd is not running, output will be incomplete.

Failed to query link bit rates: Unit dbus-org.freedesktop.network1.service not found.
Failed to query link DHCP leases: Unit dbus-org.freedesktop.network1.service not found.
2,3c2,3
< 64 bytes from 10.102.160.1: icmp_seq=1 ttl=64 time=0.337 ms
< 64 bytes from 10.102.160.1: icmp_seq=2 ttl=64 time=0.177 ms
---
> 64 bytes from 10.102.160.1: icmp_seq=1 ttl=64 time=0.174 ms
> 64 bytes from 10.102.160.1: icmp_seq=2 ttl=64 time=0.155 ms
6,7c6,7
< 2 packets transmitted, 2 received, 0% packet loss, time 1005ms
< rtt min/avg/max/mdev = 0.177/0.257/0.337/0.080 ms
---
> 2 packets transmitted, 2 received, 0% packet loss, time 1029ms
> rtt min/avg/max/mdev = 0.155/0.164/0.174/0.009 ms
11,12c11,12
< 64 bytes from 10.102.160.1: icmp_seq=1 ttl=64 time=0.130 ms
< 64 bytes from 10.102.160.1: icmp_seq=2 ttl=64 time=0.164 ms
---
> 64 bytes from 10.102.160.1: icmp_seq=1 ttl=64 time=0.136 ms
> 64 bytes from 10.102.160.1: icmp_seq=2 ttl=64 time=0.159 ms
16c16
< rtt min/avg/max/mdev = 0.130/0.147/0.164/0.017 ms
---
> rtt min/avg/max/mdev = 0.136/0.147/0.159/0.011 ms
```
```


### Additional context

_No response_
